### PR TITLE
Make namespace and proxy port configurable

### PR DESCRIPTION
**Pull request by taheris**

### CODE REVIEW

**Comment by taheris** [comment]
@doanac: thanks for the suggestions. Some fixes based on your comments

**Comment by doanac** [comment]
haven't had time to test, but it looks good to me.

**Comment by taheris** [comment]
@alexhumphreys: do you have time to take a look? Another commit was added to remove the pre-pulling of images as it is not as stable as I'd like and there are [alternative options](https://github.com/kubernetes/minikube/blob/master/docs/cache.md).

File: scripts/start.sh [hunk 1]
```diff
@@ -230,7 +221,8 @@ start_vaults() {
 
 start_weave() {
   [[ ${SKIP_WEAVE} == true ]] && return 0;
-  apply_template templates/weave
+  local version=$(${KUBECTL} version | base64 | tr -d '\n')
+  ${KUBECTL} apply -f "https://cloud.weave.works/k8s/net?k8s-version=${version}"
```
Comment (alexhumphreys) [inline scripts/start.sh]: think something like this works now:
```
curl "https://cloud.weave.works/k8s/net?k8s-version=1.9.3"
```

**Pull request merged**